src/redux/getters/getTrips.test.js

```diff
@@ -0,0 +1,95 @@
+import { mockStore } from '../mock-server/mockStore';
+import { getSortedTrips, getCurrTrip } from './getTrips';
+import { getTripTimes } from './getTripTimes';
+import { convertDate } from './convertDate';
+
+// Set up a mock store
+import configureStore from 'redux-mock-store';
+import thunkMiddleware from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+const loggerMiddleware = createLogger();
+const middlewares = [thunkMiddleware, loggerMiddleware];
+const makeStore = configureStore(middlewares);
+
+it('Has a store', () => {
+  const store = makeStore(mockStore());
+  expect(store.getState()).toEqual(mockStore());
```

Comment:
Loving this word choice. Go developers with clear names

Reply:
Truth! Hahaha!